packages/server/graphql/types/UpdateUserProfilePayload.ts

```diff
@@ -19,9 +19,9 @@ const UpdateUserProfilePayload = new GraphQLObjectType<any, GQLContext>({
     teamMembers: {
       type: new GraphQLList(new GraphQLNonNull(TeamMember)),
       description: 'The updated team member',
-      resolve: ({teamIds, userId}, _args, {dataLoader}) => {
+      resolve: ({teamIds, userId}, _args: unknown, {dataLoader}) => {
```

Comment:
nit: always nice to declare the type as early as possible, that way if we do something else with teamIds then we don't have to do multiple casts down below